birthdays-project/config/settings/base.py

```diff
@@ -68,16 +76,23 @@
     },
 ]
 
-WSGI_APPLICATION = 'birthdays.wsgi.application'
+WSGI_APPLICATION = 'config.wsgi.application'
 
 
 # Database
 # https://docs.djangoproject.com/en/1.9/ref/settings/#databases
 
 DATABASES = {
```

Comment:
I suggest to read the values from the environment. One can then configure a database with the default values or add the production/development parameters to `.env_prod` and `.env_dev`

```
DATABASES = {
    'default': {
        'ENGINE': 'django.db.backends.postgresql_psycopg2',
        'NAME': os.environ.get('DATABASE_NAME', 'birthdays'),
        'USER': os.environ.get('DATABASE_USER', 'birthdays'),
        'PASSWORD': os.environ.get('DATABASE_PASSWORD', 'birthdays'),
        'HOST': os.environ.get('DATABASE_HOST', 'localhost'),
        'PORT': 5432
    }
}
```